install-dependencies

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+# Clean object dir and return the input error
+function err(){
+  rm -rf $DEPS_GLIBC
+  exit $1
+}
+
+#
+# Applies OSX "dependencies" needed to compile the linux kernel
+#
+
+if [[ $(uname -s) = "Darwin" ]]; then
+  command -v port >/dev/null ||
+  (
+    echo "MacPorts is not installed" &&
+    echo "Please visit https://www.macports.org/ for more information" &&
+    exit 40
+  )
+
+  port -q install gsed || err $?
+
+  ln -s /opt/local/bin/gsed /usr/local/bin/sed
+
+  GLIBC_VERSION=2.23
+  GLIBC_TAR=glibc-$GLIBC_VERSION.tar.gz
+  GLIBC_URL=http://ftpmirror.gnu.org/glibc/$GLIBC_TAR
+
+  INCLUDE=/usr/local/include
```

Comment:
Could you be able to use a path under the project root instead of the global filesystem? This way we wouldn't need to have administrator permissions nor left random files in root filesystem... Later it would be possible to add a new `-I` include path location.